doc/man3/OpenSSL_version.pod

```diff
@@ -52,25 +52,25 @@ Deprecated:
 =head2 Macros
 
 The three macros B<OPENSSL_VERSION_MAJOR>, B<OPENSSL_VERSION_MINOR> and
-B<OPENSSL_VERSION_PATCH> represent the three parts of a 3 numbered version
-number, MAJOR.MINOR.PATCH.
+B<OPENSSL_VERSION_PATCH> represent the three parts of a version
+identifier, B<MAJOR.MINOR.PATCH>.
```

Comment:
In the interest of consistency, this should be `B<I<MAJOR>.I<MINOR>.I<PATCH>>`

Reply:
I wondered if that was going to get called out :)  Fixup commit pushed.